src/fluent/stdout.h

```diff
@@ -46,7 +46,7 @@ class Stdout {
   }
 
  private:
-  std::set<std::tuple<std::string>> deferred_merge_;
+  std::vector<std::tuple<std::string>> deferred_merge_;
```

Comment:
Oh my bad, I misunderstood the change. Anything that's being streamed from (like `deferred_merge_` or `buf` inside of `BufferRaInto`) can be a vector no problem. I thought you had changed the actual tuples inside of table and stuff.